src/test/java/com/thangle/api/user/UserRequestDTOMapperTest.java

```diff
@@ -0,0 +1,39 @@
+package com.thangle.api.user;
+
+import org.junit.jupiter.api.Test;
+
+import static com.thangle.fakes.UserFakes.buildUser;
+import static com.thangle.api.user.UserRequestDTOMapper.*;
+import static com.thangle.fakes.UserFakes.buildUserRequestDTO;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UserRequestDTOMapperTest {
+
+    @Test
+    void shouldToUser_OK() {
+        final var userRequestDTO = buildUserRequestDTO();
+        final var user = toUser(userRequestDTO);
+
+        assertEquals(user.getUsername(), userRequestDTO.getUsername());
```

Comment:
thông tin user là giá trị actual nên phải để ra sau

```suggestion
        assertEquals(userRequestDTO.getUsername(), user.getUsername());
```

https://junit.org/junit4/javadoc/4.8/org/junit/Assert.html#assertEquals(java.lang.Object,%20java.lang.Object)